src/NiceHashMiner/Views/Login/LoginBrowser.xaml.cs

```diff
@@ -49,6 +49,7 @@ private void Browser_Loaded(object sender, RoutedEventArgs e)
         {
             browser.Navigate(Links.LoginNHM);
             _evalTimer = new Timer((s) => { Dispatcher.Invoke(EvalTimer_Elapsed); }, null, 100, 1000);
```

Comment:
@luc1an24
give option for manual update of miner
ed2d57e
@luc1an24
use dispatcher for action